src/controllers/cart.controller.ts

```diff
@@ -0,0 +1,143 @@
+import * as Koa from 'koa'
+import * as HttpStatus from 'http-status-codes'
+import { getRepository, Repository } from 'typeorm'
+import CartPosition from '../models/cart.entity'
+import User from '../models/user.entity'
+import Dish from '../models/menu.entity'
+import Order from '../models/order.entity'
+
+export default class CartController {
+    static async getCart(ctx: Koa.Context){
+        const cartRepo: Repository<CartPosition> = getRepository(CartPosition)
+        const userRepo: Repository<User> = getRepository(User)
+        const orderRepo: Repository<Order> = getRepository(Order)
+
+        const user: User = await userRepo.findOne(ctx.params.user_id)
+        let order: Order = await orderRepo.findOne({
+            where: {
+                user: user,
```

Comment:
You should be able to write this where clause without loading the user beforehand, just by providing user id.
![image](https://user-images.githubusercontent.com/3977879/140403532-ed0fb684-80b2-4f23-abb6-7564c276c023.png)

Reply:
In general try to load the whole cart with single request. I feel that it should be possible.